chat-plugins/info.js

```diff
@@ -36,6 +36,7 @@ exports.commands = {
 		}
 
 		let buf = Chat.html`<strong class="username"><small style="display:none">${targetUser.group}</small>${targetUser.name}</strong> `;
+		if (targetUser.autoconfirmed) buf += ` <small style="color:gray">(ac)</small>`;
```

Comment:
My thought process,
Alts = an actual account a user cares about and got auto confirmed
PrevName = random accounts users don't typically care about, unregistered accounts fall in this.
This shows a users alt which should more or less be hidden from users, but again i don't care too much if this is broadcasted to regs